articles/openstack/ostack-how-static-ip.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Static IP  | UKCloud Ltd
+description: Instructions on how to create static IPs and assign them to instances
+services: openstack
+author: Bryce Nicholls
+toc_rootlink: How to
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create Static IPs for assignment to instances.
+toc_fullpath: How To/ostack-how-static-ip.md
+toc_mdlink: ostack-how-static-ip.md
+---
+
+# Creating Static IPs for assignment to instances.
```

Comment:
Change title to How to create static IPs for OpenStack instances
(no full stop at the end)